packages/sakuli-core/src/bootstrap/get-sakuli-presets.function.spec.ts

```diff
@@ -0,0 +1,28 @@
+import mockFs from "mock-fs";
+import { getSakuliPresets } from "./get-sakuli-presets.function";
+
+describe("get-sakuli-presets", () => {
+  it("should return execa package name", () => {
+    mockFs({
+      "/base": {
+        execa: {
+          "package.json":
+            '{ "name": "sakuli", "keywords": [ "sakuliPreset" ] }',
+        },
+        foo: {
+          "package.json": '{ "name": "just another package json" }',
+        },
+        "empty-dir": {},
+      },
+    });
+
+    const sakuliPresets = getSakuliPresets(["/base"]);
+
+    expect(sakuliPresets).toEqual(["sakuli"]);
+  });
```

Comment:
I find it pretty hard to follow the test case.
From the other parts of the code, in production, this code would process folder structures like `node_modules/@sakuli/someSakuliPresetProvider/package.json`, right?
What do you think about reflecting this within test case setup to better transport the use case of this function. I mean the function itself is abstract and can take/scan any location which is great, but it's hard to imagine what `/base/execa/package.json` tries to represent as such a structure is not common to the project.